transaction.go

```diff
@@ -627,6 +628,31 @@ type TransactionResult struct {
 	CollectionID  Identifier
 }
 
+// Fee returns the transaction fee that was paid by the payer account
+// in the smallest unit of Flow (10^-8).
+//
+// Error is returned if the fee events could not be parsed correctly from
+// the transaction result.
+func (t *TransactionResult) Fee() (uint64, error) {
+	var feeEvent cadence.Event
+	for _, e := range t.Events {
+		if strings.Contains(e.Type, "FlowFees.FeesDeducted") {
+			if feeEvent.Type() != nil {
+				return 0, fmt.Errorf("could not extract transaction fee")
+			}
+			feeEvent = e.Value
+		}
+	}
+
```

Comment:
fees could be disabled
```suggestion
if feeEvent == nil {
    return 0, nil
}
```